kairon/shared/data/customer_order_processor.py

```
@@ -0,0 +1,222 @@
+        )
+        order.save()
+
+        result = order.to_mongo().to_dict()
+        result["_id"] = str(result["_id"])
+        return CustomerOrderProcessor._mask_user_id(result)
+
+    @staticmethod
+    def update_order_status(bot: Text, order_id: str, new_status: str) -> Dict:
+        try:
+            order = OrderDetails.objects(bot=bot, id=ObjectId(order_id)).get()
+        except (DoesNotExist, Exception):
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)